src/eduid_common/api/schemas/csrf.py

```diff
@@ -1,9 +1,13 @@
 # -*- coding: utf-8 -*-
 
 from __future__ import absolute_import
+try:
+    import urlparse  # Python2
+except ImportError:
+    from urllib.parse import urlparse  # Python3
```

Comment:
This will not work with Python 3, since urlsplit (which is used further down) is found at urllib.parse.urlsplit